diff --git a/lib/index.js b/lib/index.js
index 2b6995e14691eb17250003989b7cf2a293e97f50..0853f7aef802f2ab8e2aaf158d76d6168a325e8f 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -38,7 +38,7 @@ const PT = __importStar(require("prop-types"));
 const classnames_1 = __importDefault(require("classnames"));
 const react_modal_1 = __importDefault(require("react-modal"));
 const nav_frontend_lukknapp_1 = __importDefault(require("nav-frontend-lukknapp"));
-require("nav-frontend-modal-style");
+require("nav-frontend-modal-style/dist/main.css");
 const cls = (className) => (0, classnames_1.default)("modal", className);
 class ModalWrapper extends React.Component {
     constructor(props) {
@@ -65,15 +65,19 @@ class ModalWrapper extends React.Component {
     render() {
         const _a = this.props, { children, closeButton, shouldCloseOnOverlayClick, contentClass } = _a, props = __rest(_a, ["children", "closeButton", "shouldCloseOnOverlayClick", "contentClass"]);
         const lukkModalLabel = "Lukk";
-        return (React.createElement(react_modal_1.default, Object.assign({}, props, { className: cls(props.className), onRequestClose: this.onRequestClose, overlayClassName: "modal__overlay", shouldCloseOnOverlayClick: true, ref: (modalRef) => {
+        return (React.createElement(react_modal_1.default, Object.assign({}, props, {
+            className: cls(props.className), onRequestClose: this.onRequestClose, overlayClassName: "modal__overlay", shouldCloseOnOverlayClick: true, ref: (modalRef) => {
                 this.modalRef = modalRef;
-            } }),
+            }
+        }),
             React.createElement("section", { className: contentClass }, children),
-            closeButton && (React.createElement(nav_frontend_lukknapp_1.default, { overstHjorne: true, className: (0, classnames_1.default)({
+            closeButton && (React.createElement(nav_frontend_lukknapp_1.default, {
+                overstHjorne: true, className: (0, classnames_1.default)({
                     "modal__lukkknapp--shake": shouldCloseOnOverlayClick,
                 }), ariaLabel: lukkModalLabel, onClick: props.onRequestClose, ref: (closeButtonRef) => {
                     this.closeButtonRef = closeButtonRef;
-                } }, lukkModalLabel))));
+                }
+            }, lukkModalLabel))));
     }
 }
 ModalWrapper.defaultProps = {
