diff --git a/lib/column.js b/lib/column.js
index 97c6411f4fcb1fd23e768ee52a556f7615689a95..4defea7501190050c7caeed1e871317257b2f44d 100644
--- a/lib/column.js
+++ b/lib/column.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -36,7 +36,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const PT = __importStar(require("prop-types"));
 const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
-require("nav-frontend-grid-style");
+require("nav-frontend-grid-style/dist/main.css");
 const cls = (className, xs, sm, md, lg) => (0, classnames_1.default)("col", className, {
     [`col-xs-${xs}`]: !!xs,
     [`col-sm-${sm}`]: !!sm,
diff --git a/lib/container.js b/lib/container.js
index 4a0e8d012c84bf2ceb308e96900e534dc404400b..f0dc385ed0198161ea1d3d3007a5e64ec56cd998 100644
--- a/lib/container.js
+++ b/lib/container.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -36,7 +36,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const PT = __importStar(require("prop-types"));
 const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
-require("nav-frontend-grid-style");
+require("nav-frontend-grid-style/dist/main.css");
 const cls = (fluid, className) => (0, classnames_1.default)(className, {
     container: fluid === false,
     "container-fluid": fluid === true,
diff --git a/lib/row.js b/lib/row.js
index 4cdb3fad2ef60e432b16a716e6847412ff89965c..43df50e3a028e49528f13f6da03c1a80617872c9 100644
--- a/lib/row.js
+++ b/lib/row.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -36,7 +36,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const PT = __importStar(require("prop-types"));
 const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
-require("nav-frontend-grid-style");
+require("nav-frontend-grid-style/dist/main.css");
 const cls = (className) => (0, classnames_1.default)("row", className, {});
 class Row extends React.Component {
     render() {
