diff --git a/lib/alertstripe.js b/lib/alertstripe.js
index 463c73f67f81696706bbff1fbd1160ba5abd2fa2..25a27e355de163dc7a14a79f91e1e8dd66807f68 100644
--- a/lib/alertstripe.js
+++ b/lib/alertstripe.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -39,7 +39,7 @@ const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
 const nav_frontend_typografi_1 = require("nav-frontend-typografi");
 const nav_frontend_ikoner_assets_1 = __importDefault(require("nav-frontend-ikoner-assets"));
-require("nav-frontend-alertstriper-style");
+require("nav-frontend-alertstriper-style/dist/main.css");
 const cls = (type, form, className) => (0, classnames_1.default)("alertstripe", className, {
     "alertstripe--info": type === "info",
     "alertstripe--suksess": type === "suksess",
