diff --git a/lib/hjelpetekst.js b/lib/hjelpetekst.js
index 3c6dc15772d0703e58b7655f10975dfc32776ae2..0c90b1c9ccdc5712f5e2ca198e7901b5a9259a2f 100644
--- a/lib/hjelpetekst.js
+++ b/lib/hjelpetekst.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -38,7 +38,7 @@ const classnames_1 = __importDefault(require("classnames"));
 const nav_frontend_js_utils_1 = require("nav-frontend-js-utils");
 const nav_frontend_ikoner_assets_1 = __importDefault(require("nav-frontend-ikoner-assets"));
 const nav_frontend_popover_1 = __importStar(require("nav-frontend-popover"));
-require("nav-frontend-hjelpetekst-style");
+require("nav-frontend-hjelpetekst-style/dist/main.css");
 class Hjelpetekst extends React.Component {
     constructor(props) {
         super(props);
