diff --git a/lib/bekreft-checkboks-panel.js b/lib/bekreft-checkboks-panel.js
index 0b89095079bd9ff41395ddff15325b176063d9a1..f66f69b391fdbef6037a6aab8da7e86bfa31ba33 100644
--- a/lib/bekreft-checkboks-panel.js
+++ b/lib/bekreft-checkboks-panel.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -29,7 +29,7 @@ const PT = __importStar(require("prop-types"));
 const nav_frontend_js_utils_1 = require("nav-frontend-js-utils");
 const skjemaelement_feilmelding_1 = __importDefault(require("./skjemaelement-feilmelding"));
 const _1 = require(".");
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 class BekreftCheckboksPanel extends React.Component {
     render() {
         const { checked, onChange, label, inputProps, className, feil, } = this.props;
diff --git a/lib/checkboks-panel-gruppe.js b/lib/checkboks-panel-gruppe.js
index c0c7109ed0144892ef42e200ec57643915b33b28..5dbbf27ec2a66eda417ab610d1928f12f63c1e53 100644
--- a/lib/checkboks-panel-gruppe.js
+++ b/lib/checkboks-panel-gruppe.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -36,7 +36,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const React = __importStar(require("react"));
 const PT = __importStar(require("prop-types"));
 const _1 = require(".");
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 const nav_frontend_js_utils_1 = require("nav-frontend-js-utils");
 const classnames_1 = __importDefault(require("classnames"));
 class CheckboksPanelGruppe extends React.Component {
diff --git a/lib/checkboks-panel.js b/lib/checkboks-panel.js
index dea4165abb43cdfbc8f6362e07dbe18b79452c31..97207626ed399db441b7cd89da1425fb880c96b8 100644
--- a/lib/checkboks-panel.js
+++ b/lib/checkboks-panel.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -36,7 +36,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
 const nav_frontend_js_utils_1 = require("nav-frontend-js-utils");
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 const _1 = require(".");
 class CheckboksPanel extends React.Component {
     constructor(props) {
diff --git a/lib/checkbox-gruppe.js b/lib/checkbox-gruppe.js
index a512be11769beb748656b89514ab16d6498726f6..070d46b1e625322e6ad053fd8562cbc3c643ef92 100644
--- a/lib/checkbox-gruppe.js
+++ b/lib/checkbox-gruppe.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -36,7 +36,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
 const _1 = require(".");
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 class CheckboxGruppe extends React.Component {
     render() {
         const _a = this.props, { children, className } = _a, other = __rest(_a, ["children", "className"]);
diff --git a/lib/checkbox.js b/lib/checkbox.js
index 57b6bd2591dcda244dea1fb855a30cc379c393f2..83f535029a7c495d6eb0d046a65c6959445f0697 100644
--- a/lib/checkbox.js
+++ b/lib/checkbox.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -40,7 +40,7 @@ const classnames_1 = __importDefault(require("classnames"));
 const nav_frontend_js_utils_1 = require("nav-frontend-js-utils");
 const skjemaelement_feilmelding_1 = __importDefault(require("./skjemaelement-feilmelding"));
 const _1 = require(".");
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 const cls = (className) => (0, classnames_1.default)("skjemaelement", className);
 const inputCls = (harFeil) => (0, classnames_1.default)("skjemaelement__input checkboks", {
     "skjemaelement__input--harFeil": harFeil,
diff --git a/lib/feiloppsummering.js b/lib/feiloppsummering.js
index 789a1965dbf08e68492307684aca99b499053ffd..8817d6f539783c0ffe2368f9d12a22caa428b2e9 100644
--- a/lib/feiloppsummering.js
+++ b/lib/feiloppsummering.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -37,7 +37,7 @@ const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
 const nav_frontend_typografi_1 = require("nav-frontend-typografi");
 const nav_frontend_lenker_1 = __importDefault(require("nav-frontend-lenker"));
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 class Feiloppsummering extends React.Component {
     render() {
         const _a = this.props, { children, className, innerRef, tittel, feil, customFeilRender } = _a, rest = __rest(_a, ["children", "className", "innerRef", "tittel", "feil", "customFeilRender"]);
diff --git a/lib/fnr-input.js b/lib/fnr-input.js
index 972494b5da91181584cfb44db8f8ccac9c967e57..9147d6c2c770b42ccf2f3a1d735ca8f88eb150bf 100644
--- a/lib/fnr-input.js
+++ b/lib/fnr-input.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -26,7 +26,7 @@ const React = __importStar(require("react"));
 const fnrvalidator_1 = require("@navikt/fnrvalidator");
 const input_1 = __importDefault(require("./input"));
 const nav_frontend_js_utils_1 = require("nav-frontend-js-utils");
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 class FnrInput extends React.Component {
     onChange(e) {
         if (this.props.onValidate) {
diff --git a/lib/input.js b/lib/input.js
index e3a943c0d698a967ee0ae21768f824d488b726b0..4b0244b5215ec5192bad69aaf09c9fc24d69b6e7 100644
--- a/lib/input.js
+++ b/lib/input.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -39,7 +39,7 @@ const classnames_1 = __importDefault(require("classnames"));
 const nav_frontend_js_utils_1 = require("nav-frontend-js-utils");
 const skjemaelement_feilmelding_1 = __importDefault(require("./skjemaelement-feilmelding"));
 const _1 = require(".");
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 const cls = (className) => (0, classnames_1.default)("skjemaelement", className);
 const inputClass = (width, className, harFeil, mini) => (0, classnames_1.default)("skjemaelement__input", className, `input--${width.toLowerCase()}`, {
     "skjemaelement__input--harFeil": harFeil,
diff --git a/lib/label.js b/lib/label.js
index 5648b836e002eb3b7181ed77843b422e62e0d470..83a1a3cade5c388b00332d7ef3ea805b7adb7ef2 100644
--- a/lib/label.js
+++ b/lib/label.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -35,7 +35,7 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
 Object.defineProperty(exports, "__esModule", { value: true });
 const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 class Label extends React.Component {
     render() {
         const _a = this.props, { children, className } = _a, other = __rest(_a, ["children", "className"]);
diff --git a/lib/radio-gruppe.js b/lib/radio-gruppe.js
index 014acbfd4a970504612445a840b250f9a5d01598..7d6883b572809cbe30f3bd905299bd5caea8e818 100644
--- a/lib/radio-gruppe.js
+++ b/lib/radio-gruppe.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -36,7 +36,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
 const _1 = require(".");
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 class RadioGruppe extends React.Component {
     render() {
         const _a = this.props, { children, className } = _a, other = __rest(_a, ["children", "className"]);
diff --git a/lib/radio-panel-gruppe.js b/lib/radio-panel-gruppe.js
index 53d1acaf6670f27f8d006eb5e0375ecf8b49a5ca..e132443947bce22853a0f26a52131d0899298b16 100644
--- a/lib/radio-panel-gruppe.js
+++ b/lib/radio-panel-gruppe.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -38,7 +38,7 @@ const PT = __importStar(require("prop-types"));
 const classnames_1 = __importDefault(require("classnames"));
 const nav_frontend_js_utils_1 = require("nav-frontend-js-utils");
 const _1 = require(".");
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 class RadioPanelGruppe extends React.Component {
     render() {
         const _a = this.props, { radios, name, legend, feil, checked, onChange, className } = _a, other = __rest(_a, ["radios", "name", "legend", "feil", "checked", "onChange", "className"]);
diff --git a/lib/radio-panel.js b/lib/radio-panel.js
index 3db1fd7640c2f2fca18f4e8752d745e48f21de92..35105e2d78d058d20a16f022cd7d3f4e7598c5ad 100644
--- a/lib/radio-panel.js
+++ b/lib/radio-panel.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -38,7 +38,7 @@ const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
 const nav_frontend_js_utils_1 = require("nav-frontend-js-utils");
 const _1 = require(".");
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 class RadioPanel extends React.Component {
     constructor(props) {
         super(props);
diff --git a/lib/radio.js b/lib/radio.js
index e070b493fbd16d37044adada6817b15632fc7f21..0d3f99292c3cf35f8f4389cdf26f4f7a4598133a 100644
--- a/lib/radio.js
+++ b/lib/radio.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -37,7 +37,7 @@ const PT = __importStar(require("prop-types"));
 const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
 const nav_frontend_js_utils_1 = require("nav-frontend-js-utils");
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 const _1 = require(".");
 const cls = (className) => (0, classnames_1.default)("skjemaelement", className);
 const inputCls = (harFeil) => (0, classnames_1.default)("skjemaelement__input radioknapp", {
diff --git a/lib/select.js b/lib/select.js
index 0d2159144e70f6b8398c646e0dad9d6fb6c304b7..f307c9d5750c35801f2cbbd600ddc3f85ae111c2 100644
--- a/lib/select.js
+++ b/lib/select.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -39,7 +39,7 @@ const classnames_1 = __importDefault(require("classnames"));
 const nav_frontend_js_utils_1 = require("nav-frontend-js-utils");
 const skjemaelement_feilmelding_1 = __importDefault(require("./skjemaelement-feilmelding"));
 const _1 = require(".");
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 const cls = (className) => (0, classnames_1.default)("skjemaelement", className);
 const inputCls = (harFeil) => (0, classnames_1.default)("skjemaelement__input", {
     "skjemaelement__input--harFeil": harFeil,
diff --git a/lib/skjema-gruppe.js b/lib/skjema-gruppe.js
index 60b350ad5bf900fa3a8005aa5b7a485d405f82b6..f06e542ca20fd1944f951d7a937ed2df5ec65508 100644
--- a/lib/skjema-gruppe.js
+++ b/lib/skjema-gruppe.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -39,7 +39,7 @@ const React = __importStar(require("react"));
 const classnames_1 = __importDefault(require("classnames"));
 const nav_frontend_js_utils_1 = require("nav-frontend-js-utils");
 const skjemaelement_feilmelding_1 = __importDefault(require("./skjemaelement-feilmelding"));
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 const cls = (className) => (0, classnames_1.default)("skjemagruppe", className);
 exports.SkjemaGruppeFeilContext = React.createContext({});
 /**
diff --git a/lib/skjemaelement-feilmelding.js b/lib/skjemaelement-feilmelding.js
index dbadd061efc6cfe5c2ae82ab61981ecdf01050b9..995f6984c90e13d1eab48893e67d51e521bba979 100644
--- a/lib/skjemaelement-feilmelding.js
+++ b/lib/skjemaelement-feilmelding.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -32,7 +32,7 @@ var __rest = (this && this.__rest) || function (s, e) {
 Object.defineProperty(exports, "__esModule", { value: true });
 const React = __importStar(require("react"));
 const nav_frontend_typografi_1 = require("nav-frontend-typografi");
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 class SkjemaelementFeilmelding extends React.Component {
     render() {
         const _a = this.props, { children } = _a, other = __rest(_a, ["children"]);
diff --git a/lib/textarea-controlled.js b/lib/textarea-controlled.js
index fc35865540e700192b867d04d5492690dd4da6e0..b5b5e1b324c025d5627c12d4c8080527ae37b9d9 100644
--- a/lib/textarea-controlled.js
+++ b/lib/textarea-controlled.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -37,7 +37,7 @@ const React = __importStar(require("react"));
 const PT = __importStar(require("prop-types"));
 const nav_frontend_js_utils_1 = require("nav-frontend-js-utils");
 const textarea_1 = __importDefault(require("./textarea"));
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 class TextareaControlled extends React.Component {
     constructor(props) {
         super(props);
diff --git a/lib/textarea.js b/lib/textarea.js
index 40b2e73f7d088ff84e1a6d6226890e1e36a6beea..021e233f20a983d09dc34bb81c2337b27d6dc8da 100644
--- a/lib/textarea.js
+++ b/lib/textarea.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -39,7 +39,7 @@ const classnames_1 = __importDefault(require("classnames"));
 const nav_frontend_js_utils_1 = require("nav-frontend-js-utils");
 const _1 = require(".");
 const skjemaelement_feilmelding_1 = __importDefault(require("./skjemaelement-feilmelding"));
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 const inputCls = (className, harFeil) => (0, classnames_1.default)(className, "skjemaelement__input textarea--medMeta", {
     "skjemaelement__input--harFeil": harFeil,
 });
@@ -93,16 +93,20 @@ class Textarea extends React.Component {
                         this.props.maxLength,
                         " tegn.")),
                     React.createElement(nav_frontend_js_utils_1.EventThrottler, { event: "resize", callback: this.updateHeight, delay: 100 },
-                        React.createElement("textarea", Object.assign({ ref: (textarea) => {
+                        React.createElement("textarea", Object.assign({
+                            ref: (textarea) => {
                                 this.tekstomrade = textarea;
                                 if (textareaRef !== undefined)
                                     textareaRef(textarea);
-                            }, onChange: onChange, className: inputCls(textareaClass, feilmelding), id: this.textareaId, name: name, style: { height: "30px" }, "aria-invalid": !!feilmelding, "aria-errormessage": feilmelding ? feilmeldingId : undefined, "aria-describedby": this.describedbyId }, other))),
+                            }, onChange: onChange, className: inputCls(textareaClass, feilmelding), id: this.textareaId, name: name, style: { height: "30px" }, "aria-invalid": !!feilmelding, "aria-errormessage": feilmelding ? feilmeldingId : undefined, "aria-describedby": this.describedbyId
+                        }, other))),
                     React.createElement(Teller, { antallTegn: antallTegn, maxLength: maxLength, tellerTekst: tellerTekst })),
                 !context.feil && !!feil && (React.createElement(skjemaelement_feilmelding_1.default, { id: feilmeldingId }, typeof feilmelding !== "boolean" && feilmelding)),
-                React.createElement("div", { className: "textareamirror", ref: (mirror) => {
+                React.createElement("div", {
+                    className: "textareamirror", ref: (mirror) => {
                         this.mirror = mirror;
-                    }, "aria-hidden": "true" })));
+                    }, "aria-hidden": "true"
+                })));
         }));
     }
 }
diff --git a/lib/toggle-gruppe.js b/lib/toggle-gruppe.js
index 2cd3f98988d8300eb86d15a7e5544f3037fdebd0..7c18e604f6e4e8615438021274f077d982265ccb 100644
--- a/lib/toggle-gruppe.js
+++ b/lib/toggle-gruppe.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -24,7 +24,7 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
 Object.defineProperty(exports, "__esModule", { value: true });
 const React = __importStar(require("react"));
 const PT = __importStar(require("prop-types"));
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 const classnames_1 = __importDefault(require("classnames"));
 class ToggleGruppe extends React.Component {
     render() {
diff --git a/lib/toggle-knapp.js b/lib/toggle-knapp.js
index 3ade08c924de20b337998325c762edc74d5979c6..3a589df74231d82fe6357eb9c7e4d92f09322877 100644
--- a/lib/toggle-knapp.js
+++ b/lib/toggle-knapp.js
@@ -1,14 +1,14 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
-    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
-}) : (function(o, m, k, k2) {
+    Object.defineProperty(o, k2, { enumerable: true, get: function () { return m[k]; } });
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || function (mod) {
@@ -25,7 +25,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const React = __importStar(require("react"));
 const PT = __importStar(require("prop-types"));
 const nav_frontend_js_utils_1 = require("nav-frontend-js-utils");
-require("nav-frontend-skjema-style");
+require("nav-frontend-skjema-style/dist/main.css");
 const classnames_1 = __importDefault(require("classnames"));
 class ToggleKnapp extends React.Component {
     render() {
